tests/engine/test_debit_note_intervals.py

```diff
@@ -107,7 +106,7 @@ def matches(reason, _regex):
             _is_none,
             _is_none,
         ),
-        # MAP active and we have received n-th debit note earlier than expected
+        # MAP active and we have received n-th+1 debit note earlier than expected
```

Comment:
and here

Reply:
Seems like this one was tested at +1 already